packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -83,9 +95,10 @@ class Popover extends React.Component {
   };
 
   static defaultProps = {
+    closeOnScrolled: true,
     component: "span",
     fullWidth: false,
-    isOpen: true,
+    isOpen: false,
```

Comment:
@stredger changing this back to `true` fixes the Invite Students ( `web/components/Students/Header` ) Popover issue.

However during this investigation, noticed that when this is changed back to true, Popover is constantly re-rendered because of change on `L151`, which removed `prevProps` check.

Next steps:
* I'll open a PR to change isOpen back to `true` by default
* I'll create a new ticket to investigate how to not have Popover re-rendering endlessly while it is open, while also still having it appear in the correct position on first render in Timeline/Milestone tooltips